tox.ini

```diff
@@ -22,3 +22,9 @@ commands =
 [testenv:flake8]
 deps = flake8
 commands = flake8 ./tox_pip_version
+
+[testenv:pip18_1_env]
+setenv =
+  TOX_PIP_VERSION=18.1
+commands =
+  bash ./tests/check-pip-version.sh 18.1
```

Comment:
That makes sense. In that case, I should change the travis configs to call `make test` instead of calling tox directly.

(Also, as the repo owner, you're going to have to be the one to enable that integration on github)

Reply:
Yep, travis is working now. The github checks are pretty nice, https://github.com/pglass/tox-pip-version/pull/2/checks